hudi-client/src/main/java/org/apache/hudi/table/RollbackExecutor.java

```diff
@@ -217,7 +216,7 @@ private HoodieRollbackStat mergeRollbackStat(HoodieRollbackStat stat1, HoodieRol
 
   private Map<HeaderMetadataType, String> generateHeader(String commit) {
     // generate metadata
-    Map<HeaderMetadataType, String> header = Maps.newHashMap();
+    Map<HeaderMetadataType, String> header = new HashMap<>();
```

Comment:
Collections.emptyMap() works? Similarly, if we can pull this into our own `CollectionUtils` class, that would be useful to change later on?

Reply:
> Collections.emptyMap() works? Similarly, if we can pull this into our own `CollectionUtils` class, that would be useful to change later on?

Hi @vinothchandar, `Collections.emptyMap()`, `Collections.emptyList()`, `Collections.emptySet()` don't support put / add /remove operations.

Reply:
good call. Still moving this in some form to a common class would be good.